pallets/relayer/src/lib.rs

```diff
@@ -0,0 +1,391 @@
+// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
+// Copyright (c) 2020 Polymath
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+//! # Simple Relayer Module
+//!
+//! The Simple Relayer module provides extrinsics for subsidising of
+//! both transaction fees as well as protocol fees.
+//!
+//! ## Overview
+//!
+//! The Simple Relayer module provides functions for:
+//!
+//! - Adding or removing a subsidiser for another user's key.
+//! - Managing how much POLYX can be used by a user key to pay
+//!   transaction/protocol fees.
+//!
+//! ## Interface
+//!
+//! ### Dispatchable Functions
+//!
+//! - `set_paying_key` - creates an authorization to allow a `user_key`
+//!   to accept a `paying_key` as their subsidiser.
+//! - `accept_paying_key` - accepts a `paying_key` authorization.
+//! - `remove_paying_key` - removes the `paying_key` from a `user_key`.
+//! - `update_polyx_limit` - updates the available POLYX for a `user_key`.
```

Comment:
```suggestion
//! - `set_paying_key` creates an authorization to allow a `user_key`
//!   to accept a `paying_key` as their subsidiser.
//! - `accept_paying_key` accepts a `paying_key` authorization.
//! - `remove_paying_key` removes the `paying_key` from a `user_key`.
//! - `update_polyx_limit` updates the available POLYX for a `user_key`.
```

Prefer normal sentences (see e.g., the corporate actions pallet).